tests/test_dialects.py

```diff
@@ -2086,7 +2086,7 @@ def test_snowflake(self):
         )
         self.validate(
             "SELECT TO_TIMESTAMP(1659981729000, 3)",
-            "SELECT TO_TIMESTAMP(1659981729)",
+            "SELECT TO_TIMESTAMP(1659981729000, 3)",
```

Comment:
this test is wrong, it should be "micros"

Reply:
I don't see what's wrong here? We're reading and writing back as snowflake, which wants the numerical argument and not a string like "micros"

Reply:
ah ok, can you write a test where you actually test micros then? i guess identity=True by default